### ISSUE TITLE
LEDs and display keep turning on and off

### ISSUE THREAD

**Issue by GoldenDuckie** [duckietown/duckietown-shell-commands#334]
The LEDs and the display on my duckiebot (DB21M) keep turning on and off. They turn off for about 10 seconds and then turn on again for 10 seconds, this causes the virtual joystick code to not function properly as when I press the forward key and robot the turns off, it keeps going forward without any command. The green LEDs on the jetson do not turn off.

**Comment by tanij** [comment]
Sounds like camera cable misplugged. Please open a question on Stack Overflow (you might find the answer there already).

**Comment by GoldenDuckie** [comment]
> Sounds like camera cable misplugged. Please open a question on Stack Overflow (you might find the answer there already).

Sadly, this did not work. I checked if all the wires were connected in the right place and they were. For the camera cable, isnt there only way to plug the camera cable as there is only 1 port to plug the camera cable

Edit: Turns out that the issue was that the camera cable was actually misplugged. It was plugged in the wrong way around and i switched and it worked fine
